chainer_setup_build.py

```diff
@@ -241,3 +289,63 @@ def get_ext_modules():
 
     check_extensions(extensions)
     return extensions
+
+
+def _nvcc_gencode_options():
+    """Returns NVCC gencode options generated from NVCC command line help."""
+    help_string = subprocess.check_output(
+        ['nvcc', '--help']).decode('ascii').replace('\n', '')
+
+    arch_options = re.findall("'(compute_\d{2})'", help_string)
+    arch_options = sorted(list(set(arch_options)))
+    arch_options = list(filter(lambda x: x >= "compute_30", arch_options))
```

Comment:
Fixed to use single quotes!

If `compute_100` appears accidentally somewhere in `nvcc` help string, it is skipped by the regular expression `'(compute_\d{2})'` that contains enclosing single quotes so the current code works correctly. On the other hand, when nVidia will provide `compute_100` in the future, it will not work. You mean the former case?